deploy/createTypesPackages.mjs

```diff
@@ -10,7 +10,34 @@ export const packages = [
     readme: "./readmes/web.md",
     files: [
       { from: "../generated/dom.generated.d.ts", to: "index.d.ts" },
-      { from: "../generated/dom.iterable.generated.d.ts", to: "index.iterable.d.ts" }
+      { from: "../generated/dom.iterable.generated.d.ts", to: "index.iterable.d.ts" },
+    ],
+  },
+  {
+    name: "@types/serviceworker",
+    description: "Types for the global scope of Service Workers",
+    readme: "./readmes/serviceworker.md",
+    files: [
+      { from: "../generated/serviceworker.generated.d.ts", to: "index.d.ts" },
+      { from: "../generated/serviceworker.iterable.generated.d.ts", to: "index.iterable.d.ts" },
```

Comment:
While you are at it: how can `index.iterable.d.ts` be referenced? 👀 Should the step be added to readme?

Reply:
Hrm, I thought it would be automatically be included in all builds because it lives in @types and we read all the files in there. Haven't _actually checked though.

Reply:
Somehow it doesn't work like that in my environment:

Reply:
Agree, the build pipeline will probably need to add something like `/// <reference path="..." />` to the `index.dt.s`

A user can probably get these imported into their globals via `import "web/index.iterable"` but that's weaksauce

Reply:
But that would mean packages will require es6+ 🤔

Reply:
I'm still figuring out how to have it happen automatically `/// <reference types="web/index.iterable" />` would also work in any target

this pattern may be a good replacement for `@types/ie11` too.

Reply:
Yeah, since it's probably time to concentrate on es6+ 😁